set-kubectl/README.md

```diff
@@ -0,0 +1,11 @@
+# Set Kubernetes CLI
+
+[kubectl](https://github.com/Azure/setup-kubectl)  is used to set up the kubectl command-line tool in your GitHub Actions workflow.kubectl is the Kubernetes command-line tool that allows you to run commands against Kubernetes clusters. By using this action, you can automate the process of installing and configuring kubectl in your CI/CD pipelines.
```

Comment:
Add that it guarantees the kubectl version is the same as the Kubernetes cluster version